catkin_virtualenv/src/dh_virtualenv/deployment.py

```diff
@@ -87,7 +86,6 @@ def __init__(self,
         self.pip_version = pip_version
         self.extra_virtualenv_arg = extra_virtualenv_arg
         self.verbose = verbose
-        self.setuptools = setuptools
```

Comment:
this option is not really relevant - either way you end up with setuptools in the virtualenv